index.html

```diff
@@ -1,544 +1,627 @@
+                  <select id="subject" name="subject">
+                    <option>Select dropdown</option>
+                    <option>With options</option>
+                  </select>
                 </div>
               </div>
+            </fieldset>
+
+            <fieldset class="field">
+              <label class="label" for="message">Message</label>
+              <div class="control">
+                <textarea id="message" class="textarea" placeholder="Enter your message"
+                  name="message"></textarea>
+              </div>
+            </fieldset>
+
+            <fieldset class="field">
+              <label class="checkbox">
+                <input id="terms" type="checkbox" name="terms">
+                I agree to the <a href="#">terms and conditions</a>
+              </label>
+            </fieldset>
+
+            <fieldset class="field">
+              <legend class="label">Subscribe to our newsletters?</legend>
+              <div class="control">
+                <label class="radio">
+                  <input id="yes" type="radio" name="yesno" value="yes">
+                  Yes
+                </label>
+                <label class="radio">
+                  <input id="no" type="radio" name="yesno" value="no">
+                  No
+                </label>
+              </div>
+            </fieldset>
+
+            <fieldset class="field is-grouped">
+              <div class="control">
+                <button type="submit" class="button is-link">Submit</button>
+              </div>
+              <div class="control">
+                <button type="button" class="button is-text">Cancel</button>
+              </div>
+            </fieldset>
+          </form>
+        </div>
+      </section>
+
+      <section class="section">
+        <div class="container">
+          <div class="section-heading has-text-centered">
+            <h2 class="title is-2">Forum</h2>
+            <h3 class="subtitle is-5">Discuss any topic with no leaving this
+              page</h3>
+          </div>
+          <div class="columns">
+            <div class="column is-3">
+              <a href="/todo"
+                class="button is-primary is-block is-alt is-large">New Post</a>
             </div>
+            <div class="column is-9">
+              <div class="box content">
+                <article class="post">
+                  <h1>How do you center a button in a box?</h1>
+                  <div class="media">
+                    <div class="media-left">
+                      <p class="image is-32x32">
+                        <img
+                          src="https://source.unsplash.com/random/64x64/?sig=300"
+                          alt>
+                      </p>
+                    </div>
+                    <div class="media-content">
+                      <div class="content">
+                        <p>
+                          <a href="/todo" tabindex="-1">@jsmith</a> replied 34
```

Comment:
Why did you put tabindex=-1? The element is not accessible via the keyboard

Reply:
It was accessible by default, as it's a link.
And it's no make sense to navigate with keyboard to author of forum topic without link to that topic. But it could be good to wrap heading with such link :)

Reply:
I would assume that it depends on the context. This link could link to person profile. But in case we don't have such page or you think that it is not a link, then it should be different from other links, otherwise it confuses users